segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/AutoScalerConfig.java

```diff
@@ -131,4 +132,28 @@ private AutoScalerConfig(TypedProperties properties) throws ConfigurationExcepti
     public static ConfigBuilder<AutoScalerConfig> builder() {
         return new ConfigBuilder<>(COMPONENT_CODE, AutoScalerConfig::new);
     }
+
+    @Override
+    public String toString() {
+        // Note: We don't use Lombok @ToString to automatically generate an implementation of this method,
+        // in order to avoid returning a string containing sensitive security configuration.
+
+        return new StringBuilder("AutoScalerConfig(")
```

Comment:
According to codecov, this is the code that is causing a low code coverage mark.

Reply:
Sure, Flavio. I've addressed this by adding a few tests, in the latest commit that I just pushed.